packages/docs/content/patterns/utility-actions.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Utility Actions
+intro: Utility actions for a webpage including print, email, RSS and more.
+---
+
+import {
+  DownloadIcon,
+  EmailIcon,
+  LinkIcon,
+  PrintIcon,
+  RssIcon,
+  WhatsNewIcon,
+} from '@cmsgov/design-system';
+
+## Guidance
+
+### Examples
+
+<EmbeddedExample>
+  <div className="ds-l-row">
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <PrintIcon className="ds-u-margin--05" viewBox="0 0 32 32" />
+        Print
+      </button>
+    </div>
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <EmailIcon className="ds-u-margin--05" viewBox="0 0 32 32" />
+        Email
+      </button>
+    </div>
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <RssIcon className="ds-u-margin--05" viewBox="0 0 32 32" />
+        RSS
+      </button>
+    </div>
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <LinkIcon className="ds-u-margin--05" viewBox="0 0 32 32" />
+        Link
+      </button>
+    </div>
+    <div className="ds-l-col--6">
+      <button className="ds-c-button ds-c-button--ghost">
+        <DownloadIcon className="ds-u-margin--05" viewBox="0 0 18 18" />
+        Download
+      </button>
```

Comment:
Looking at the Figma designs, the "small" size of the "ghost" button is used in the example. And it looks like there's about 8px between the icon and the text, so I think you can swap out the 4px margin around the icon for right margin of 8px. It also doesn't look like passing in the viewbox arguments are necessary. Feel free to look over this suggestion and adjust or ignore accordingly!

```suggestion
      <Button variation='ghost' size='small'>
        <DownloadIcon className="ds-u-margin-right--1" />
        Download
      </Button>
```